generator/internal/stub_factory_rest_generator.cc

```diff
@@ -91,7 +92,8 @@ std::shared_ptr<$stub_rest_class_name$>
 CreateDefault$stub_rest_class_name$(Options const& options) {
   Options opts = options;
   if (!opts.has<UnifiedCredentialsOption>()) {
-    opts.set<UnifiedCredentialsOption>(MakeGoogleDefaultCredentials(options));
+    opts.set<UnifiedCredentialsOption>(
```

Comment:
Seems like all the code generated here could go into a common function in `google::cloud::internal` or `google::cloud::rest_internal`.  There is nothing specific to this stub from lines 93 to 107.

Reply:
Ack